### ISSUE TITLE
[a11y] SubPlat payment page clickable elements must be focusable

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#11370]
The SubPlat payment page has clickable elements which must be focusable and should have interactive semantics in order to meet accessibility standards. 

Elements that need updating:

* graphic "PayPal"
* entry "Name as it appears on your card Full Name"
* label "Your card Secure card payment input frame"
* statictext "." DOMNode: ::before

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXA-4340)

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

Rebecca Billings , Can you clarify what you mean by focusable and interactive semantics?

What is the current behavior, and what behavior are you expecting for each of the elements that need updating?

**Comment by data-sync-user** [comment]
➤ Ivo Plamenac commented:

Rebecca Billings can you provide more details on which tools are used for testing accessibility?

**Comment by data-sync-user** [comment]
➤ Rebecca Billings commented:

The error is what is given by the accessibility inspector in Firefox. \[https://developer.mozilla.org/en-US/docs/Tools/Accessibility_inspector ( https://developer.mozilla.org/en-US/docs/Tools/Accessibility_inspector|smart-link ) ]

The issue is that each element on the page needs to be able to be navigated via keyboard, otherwise people using a11y tools cannot access the full content of the page.

**Comment by cknowles-admin** [comment]
FxA is no longer synchronizing all issues between Jira and Github.  We are closing open issues and will selectively synchronize in the future.
